content/english/useless-box/the-components.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "The Components"
+description: ""
+draft: true
+weight: 1
+---
+
+The components we will use for the project are listed below. Make sure you have all of these before starting: 
+
+1. 2 AA batteries
```

Comment:
Images for all these would be nice

Reply:
Sounds good, we'll be adding some pictures on this, if possible in an upcoming PR.

Reply:
If it'll be separate, please add a comment saying to do this

```
<!-- TODO: add images -->
```